.golangci.yml

```diff
@@ -1,11 +1,23 @@
 run:
   timeout: 5m
 
+linters-settings:
+  golint:
+    min-confidence: 0
+  misspell:
+    locale: US
+
 linters:
   disable-all: true
   enable:
     - gofmt
     - golint
+    - errcheck
     # TODO(oxddr): enable more checkers
     # - deadcode
-    # - errcheck
+    # - misspell
```

Comment:
I see you're enabling it above - why you added it here?

Reply:
That is just a configuration being added, I will re-enable it once all the errcheck issues are solved.